Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -127,6 +130,54 @@ public async Task<IActionResult> SaveCampaign(string accountName, int campaignId
             return new OkObjectResult($"La campaña '{campaignId}' del usuario '{accountName}' se guardó exitosamente ");
         }
 
+        [Authorize(Policies.OwnResourceOrSuperUser)]
+        [HttpPost("/accounts/{accountName}/campaigns/{campaignId}/content/from-template/{templateId}")]
+        public async Task<IActionResult> CreateCampaignContentFromTemplate(string accountName, int campaignId, int templateId)
+        {
+            var campaignState = await _campaignContentRepository.GetCampaignState(accountName, campaignId);
+            if (!ValidateCampaignStateToUpdate(campaignState, out var error))
+            {
+                return error;
+            }
+
+            var templateData = await _templateRepository.GetTemplate(accountName, templateId);
+            if (templateData == null)
+            {
+                return new NotFoundObjectResult(new ProblemDetails()
+                {
+                    Title = $@"The template was no found",
+                    Detail = $@"The template not exists or Inactive"
+                });
+            }
+            if (templateData.EditorType != EditorTypeUnlayer)
+            {
+                return new BadRequestObjectResult(new ProblemDetails()
+                {
+                    Title = $@"The template cannot open",
+                    Detail = $@"The template exist but is not unlayer template"
+                });
+            }
+
+            var htmlDocument = await ExtractHtmlDomFromCampaignContent(accountName, templateData.HtmlCode);
+            var head = htmlDocument.GetHeadContent();
+            var content = htmlDocument.GetDopplerContent();
+            var fieldIds = htmlDocument.GetFieldIds();
+            var trackableUrls = htmlDocument.GetTrackableUrls();
+
+            // TODO: Validate if it's possible to delete PreviewImage property from BaseHtmlContentData,
+            // because it's already in campaignContent
+            // See it on: https://github.com/FromDoppler/doppler-html-editor-api/pull/111#discussion_r870681998
+            BaseHtmlContentData baseHtmlContent = new UnlayerContentData(
+                    HtmlContent: content,
+                    HtmlHead: head,
+                    Meta: templateData.Meta,
+                    PreviewImage: templateData.PreviewImage);
+
+            await SaveCampaignContent(baseHtmlContent, fieldIds, trackableUrls, campaignState);
```

Comment:
Hay una cosa que falta, creo que se puede dejar para después, pero se podría poner al menos un _// TODO_.

Cuando se crea un contenido en base a un template, en la tabla contenido se agrega el id del template. Ahora estamos perdiendo esa información.

```suggestion
            // TODO: Save templateId reference with the content
            await SaveCampaignContent(baseHtmlContent, fieldIds, trackableUrls, campaignState);
```